addons/test_mail/static/tests/activity_tests.js

```diff
@@ -205,7 +205,7 @@ QUnit.test('activity view: batch send mail on activity', async function (assert)
     ]);
 });
 
-QUnit.test('activity view: activity widget', async function (assert) {
+QUnit.skip('activity view: activity widget', async function (assert) {
```

Comment:
this one is only failing when another test fails before, no need to skip it

Reply:
Yes me noticed that :). Could be great to understand why and fix that too